src/components/ui/ScrollToTopButton/ScrollToTopButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { CaretUp as CareUpIcon } from 'phosphor-react'
+
+import Button from '../Button'
+
+interface ScrollToTopButtonProps {
+  /**
+   * Button copy.
+   * @default 'Scroll to top'
+   */
+  text?: string
+}
+
+function ScrollToTopButton({ text = 'Scroll to top' }: ScrollToTopButtonProps) {
+  return (
+    <Button
+      variant="secondary"
+      icon={<CareUpIcon size={16} />}
+      iconPosition="left"
```

Comment:
Totally not a blocker, but just wondering if maybe it could be passed as prop too, what do you think?

Reply:
[Done](https://github.com/vtex-sites/base.store/commit/9ace211005adcf3c0898130eef3c8c98ec8d25b5)!